### DISCUSSION TITLE
Dynamic og image

### DISCUSSION THREAD

**Discussion by harshraj8843** [codinasion/codinasion-programme#1337] [Ideas]
Hey @codinasion/codinasion-programme, @codinasion/codinasion-dsa and everyone

We are working on `dynamic og image generator` for codinasion

Everyone is welcome to provide their valuable feedback and suggestion 💬 

The Source Code will be available soon 🕦 

## Why ❔ 

- create og image with latest data
- reduce repo size ( by removing static og images )
- easier for fork-clone repo(s) ( by reducing the size of repo )
  - Size of [codinasion/codinasion-programme](https://github.com/codinasion/codinasion-programme) with static og images
  ![repo-size](https://user-images.githubusercontent.com/54644599/175821333-84f6dc83-39ef-4ec2-ae68-2e9ff1bca3d3.png)

## Progress 👨🏻‍💻 

For starter, we are focused only on [codinasion/codinasion-programme](https://github.com/codinasion/codinasion-programme), more repos will be added soon

## Sample Image 🖼️ 

Here is a sample image generated

![sample-image](https://user-images.githubusercontent.com/54644599/175821284-9c695e87-172a-4840-90c3-1588b6c0db63.png)

## Reference 🔗 

- https://github.blog/2021-06-22-framework-building-open-graph-images/
- https://github.com/vercel/og-image

**Comment by victoriacheng15** [comment]
I like this. Look simple and elegant. :D

**Comment by harshraj8843** [comment]
Source code is public now 🎉 🎉 🎉 

Repo Link : https://github.com/codinasion/codinasion-og

We would appreciate hearing your thoughts on improving this 😎

**Reply by PrajwalBorkar** [reply]
Great Work @harshraj8843 👌

**Comment by harshraj8843** [comment]
https://github.com/codinasion/codinasion-og/issues/1

Tried to ` fix ` it, but unable to do so 🙁 

You are welcome to raise a PR if you could ` fix ` it 🤗 

Thanks 👍🏻
